cpp/src/command_classes/CentralScene.cpp

```diff
@@ -80,7 +99,7 @@ bool CentralScene::RequestState
 {
 	Log::Write(LogLevel_Info, GetNodeId(), "CentralScene RequestState: %d", _requestFlags);
 	bool requests = false;
-	if( ( _requestFlags & RequestFlag_AfterMark ))
+	if( ( _requestFlags & RequestFlag_Dynamic ))
```

Comment:
This should be:
if( (_requestFlags & RequestFlag_Static) && HasStaticRequest( StaticRequest_Values ) )